routing/http/client/client.go

```diff
@@ -83,6 +90,37 @@ func WithIdentity(identity crypto.PrivKey) Option {
 	}
 }
 
+// WithDisabledLocalFiltering disables local filtering of the results.
+// This should be used for delegated routing servers that already implement filtering
+func WithDisabledLocalFiltering() Option {
```

Comment:
Maybe better to have this take a boolean, so it can be blindly passed a value from a config file instead of being called conditionally?